src/main/java/HypeBot.java

```diff
@@ -0,0 +1,230 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class HypeBot {
+    private static final String BUFFER_LINE = "______________________________________________________________________\n";
+    private static final ArrayList<Task> TASKS = new ArrayList<>();
```

Comment:
Maybe tasks should be in lowercase since it is constantly changing? Like the content is changing even though the pointer remains the same? So might not be classified fully as a CONSTANT.